services/userService.js

```diff
@@ -0,0 +1,73 @@
+const userDao = require("../models/userDao");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const {
+  emailValidation,
+  pwValidation,
+  phoneNumValidation,
+  accountExistanceCheck,
+} = require("../utils/user_validation");
+
+const createUser = async (
+  account,
+  password,
+  email,
+  name,
+  phoneNum,
+  birthdate,
+  gender
+) => {
+  const accountExistanceCheck = async (account) => {
+    const [accountExistanceCheck] = await userDao.login(account);
+
+    if (accountExistanceCheck) {
+      const error = new Error("THIS ID EXISTS ALREADY!!");
+      error.statusCode = 400;
+      throw error;
+    }
+  };
```

Comment:
함수로 묶어주지 않고 다음과 같이 작성할 수 있을 것 같습니다.
또한 Dao에서 조회하고 있는 유저의 정보는 곧 데이터베이스에 저장된 유저의 정보입니다.
따라서 변수명도 `user` (user라는 변수가 사용되어야 할 경우가 있다면, `userInfo`)라고 수정하는 것도 좋을 것 같습니다.
```suggestion
  const [ user ] = await userDao.login(account);

  if (user) {
    const error = new Error("THIS ID EXISTS ALREADY!!");
    error.statusCode = 400;
    throw error;
  }
```